### ISSUE TITLE
Too long a argument line in windows with sbt-typescript 0.3.4

### ISSUE THREAD

**Issue by ndimas** [ArpNetworking/sbt-typescript#60]
Seems that sbt-typescript produces too big of a command line argument for windows to handle.

Is anybody else in the same situation?

Here is the stacktrace

[info] TypeScript compiling on 102 source(s)

akka.actor.ActorInitializationException: exception during creation
	at akka.actor.ActorInitializationException$.apply(Actor.scala:166)
	at akka.actor.ActorCell.create(ActorCell.scala:596)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:456)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:478)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:263)
	at akka.dispatch.Mailbox.run(Mailbox.scala:219)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at akka.util.Reflect$.instantiate(Reflect.scala:66)
	at akka.actor.ArgsReflectConstructor.produce(Props.scala:352)
	at akka.actor.Props.newActor(Props.scala:252)
	at akka.actor.ActorCell.newActor(ActorCell.scala:552)
	at akka.actor.ActorCell.create(ActorCell.scala:578)
	... 7 more
**Caused by: java.io.IOException: Cannot run program "node": CreateProcess error=206, The filename or extension is too long**
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at akka.contrib.process.BlockingProcess.<init>(BlockingProcess.scala:55)
	... 16 more
Caused by: java.io.IOException: CreateProcess error=206, The filename or extension is too long
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
	at java.lang.ProcessImpl.start(ProcessImpl.java:137)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 17 more

**Comment by thedailycommute** [comment]
I started getting this _exact_ same message yesterday. After a whole bunch of false leads, I finally realised the problem is related to the overall length of the paths generated when typescript files are compiled. I ended up having to shorten some folder names, and in others remove a level of folders completely.

The problem may manifest itself on one machine, but not another, depending on the full path to the project root folder. One of my colleagues keeps his projects in a single folder at the root of drive C. I keep mine in a hierarchy of project folders, 3 or 4 levels deep. Those extra paths were enough to cause me problems later on.

Hope this helps anyone else that might hit this (very vague, at least while trying to figure out what changed!) error message.
